rpcserver.go

```diff
@@ -516,12 +516,22 @@ func (r *rpcServer) FinalizeBatch(_ context.Context,
 	req *mintrpc.FinalizeBatchRequest) (*mintrpc.FinalizeBatchResponse,
 	error) {
 
+	feeRateOpt := fn.None[chainfee.SatPerKWeight]()
 	feeRate, err := checkFeeRateSanity(req.FeeRate)
 	if err != nil {
 		return nil, err
 	}
 
-	batch, err := r.cfg.AssetMinter.FinalizeBatch(feeRate)
+	if feeRate != nil {
+		feeRateOpt = fn.Some(*feeRate)
+	}
+
+	// TODO(jhb): Add tapscript tree as an arg in FinalizeBatchRequest.
```

Comment:
Comment to mark TODO as blocking before merge.

Reply:
TODO has been implemented in this PR now.